MANIFEST.in

```diff
@@ -1,5 +1,4 @@
 include LICENSE
-recursive-include LICENSES
+recursive-include LICENSES *
```

Comment:
noticed that the third-party licenses in `LICENSES/` weren't actually getting included! Because `recursive-include` requires a pattern to match.